client/my-sites/site-settings/feed-settings/index.jsx

```diff
@@ -49,7 +49,7 @@ class FeedSettings extends Component {
 					isSaving={ isSavingSettings }
 					onButtonClick={ handleSubmitForm }
 					showButton
-					title={ translate( 'Feed Settings' ) }
+					title={ translate( 'Feed settings' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.